### DISCUSSION TITLE
[QuickBackLinkV2] Append citation information to backlink

### DISCUSSION THREAD

**Discussion by windingwind** [windingwind/zotero-better-notes#776] [Note Templates]
### Description

Append citation information to backlink.

See https://github.com/windingwind/zotero-better-notes/issues/723

### Screenshots

<img width="286" alt="image" src="https://github.com/windingwind/zotero-better-notes/assets/33902321/1c79ecc8-1af1-411a-af57-d4c9406e5059">

### Test on Zotero version

7.0.0-beta.47+f012a348a

### Test on Better Notes version

1.1.4-31

### Template Type

QuickBackLink

### Template Share Code

```yaml
# This template is specifically for importing/sharing, using better 
# notes 'import from clipboard': copy the content and
# goto Zotero menu bar, click Edit->New Template from Clipboard.  
# Do not copy-paste this to better notes template editor directly.
name: "[QuickBackLinkV2]"
content: |-
  <p>
    Referred in
    <a href="${link}">
      ${linkText}
    </a>
    ${{
  	const subTopItem = Zotero.Items.getTopLevel([subNoteItem])[0];
  	if (!subTopItem.isRegularItem()) {
  		return "";
  	}
  	return Zotero.BetterNotes.api.convert.item2citation(
  		[subTopItem.id],
  		[{prefix: `${subTopItem.getField("publicationTitle")}; "${subTopItem.getField("title")}"; `}]
  	);
    }}$
  </p>
```

**Comment by DonnaCoxBaker** [comment]
Wonderful! Thanks so much!

**Comment by Fontanapink** [comment]
I am trying to use your template but I get the following error: `Template [QuickBackLinkV2] Error: SyntaxError: missing : after property id`

**Reply by windingwind** [reply]
Test on Zotero version
7.0.0-beta.47+f012a348a

Test on Better Notes version
1.1.4-31

Update your plugin & Zotero.

If you are on Zotero 6, wait for the official Zotero 7 release or install Zotero 7 beta.

**Reply by Fontanapink** [reply]
OHHHHH. 

How do I get Better Notes version 1.1.4? Mine is 1.0.4 and I can't see any other release in github

**Reply by windingwind** [reply]
https://github.com/windingwind/zotero-better-notes#-install

**Reply by Fontanapink** [reply]
Thank you for your patience =)
